README.rst

```diff
@@ -10,6 +10,8 @@ Research repository for the vivarium_nih_us_cvd project.
 Installation
 ------------
 
+Installation requires a Python version between 3.9 and 3.11.
```

Comment:
This got updated from "between foo and bar" to "between 3.9 and 3.11" in my test